autoPyTorch/data/tabular_feature_validator.py

```diff
@@ -555,10 +543,8 @@ def has_object_columns(
         True if the DataFrame dtypes contain an object column, False
         otherwise.
     """
-    object_columns_indicator = [True if pd.api.types.is_object_dtype(feature_type) else False
-                                for feature_type in feature_types]
-
-    if True in object_columns_indicator:
-        return True
-    else:
-        return False
+    for feature_type in feature_types:
+        if pd.api.types.is_object_dtype(feature_type):
+            return True
+        else:
+            return False
```

Comment:
Probably you already read my comment beforehand, but you cannot detect the case like `feature_types = [int, object]`.

Reply:
Yes, as soon as you pointed it out I noticed the bug that I introduced with the typo. On another note, the code now without that typo does not seem faster than your version, I guess numpy does have its perks.